CHANGES.rst

```diff
@@ -123,6 +123,8 @@ general
 
 - bump dependency to use ``stcal 1.9.0`` [#8808]
 
+- Remove upper pin on ``numpy`` [#8718]
+
```

Comment:
```suggestion
```

now that #8671 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Remove upper pin on ``numpy``" > changes/8718.general.rst
```

(new PRs will include instructions on how to do this)